frontend/src/components/MyTextField/MyTextField.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import * as PropTypes from "prop-types";
+import { useField } from "formik";
+import TextField from "@material-ui/core/TextField";
+
+const MyTextField = ({ label, type, ...props }) => {
```

Comment:
The name is very ambiguous. Better name it something more descriptive like `TextFieldWrapper` or maybe you can import the material TextField with a different name and name this component simply `TextField`

Reply:
Other than that, everything looks pretty solid.